.github/workflows/forest.yml

```diff
@@ -358,14 +358,11 @@ jobs:
           /cache"
 
           sudo chmod -R 755 $FIL_PROOFS_PARAMETER_CACHE
-  # Temporarily disabling the job as it's flaky
-  # Tracking issue: https://github.com/ChainSafe/forest/issues/3926
   offline-rpc-check:
     needs:
       - build-ubuntu
     name: Offline RPC check
     runs-on: ubuntu-latest
-    if: false
```

Comment:
Could you manually re-run the job for, say, 10 times and make sure it's no longer flaky?

Reply:
sure